crates/flux-refineck/src/sigs/default.rs

```diff
@@ -166,3 +167,17 @@ pub(crate) fn mk_not() -> (mir::UnOp, Sig<1>) {
     define_btys! { let bool = BaseTy::Bool; }
     (mir::UnOp::Not, s!(fn(a: bool) -> bool[a.not()]))
 }
+
+pub(crate) fn mk_lnot() -> impl IntoIterator<Item = (mir::UnOp, Sig<1>)> {
+    let int_lnots = INT_TYS.map(|int_ty| {
+        define_btys! { let Int = BaseTy::Int(int_ty); };
+        (mir::UnOp::Not, s!(fn(a: Int) -> Int[E::tt()]))
```

Comment:
This needs to be

```suggestion
        (mir::UnOp::Not, s!(fn(a: Int) -> Int{ v: E::tt() }))
```


`Int[E::tt()]` would mean the integer is indexed by a `bool` which is not well-formed.